provisioners/file/var/spool/cron/tabs/jenkins

```diff
@@ -0,0 +1,4 @@
+# DO NOT EDIT THIS FILE - edit the master and reinstall.
+# (C installed on Thu Apr  5 21:42:22 2018)
+# (Cronie version 4.2)
+43 * * * * /bin/bash /home/ec2-user/bin/limit-logs
```

Comment:
This really shouldn't live in `ec2-user`'s home now that they don't run it anymore…

Reply:
What other nice location do we have for it ?

Reply:
Some place in `/data/jenkins` I guess? `/usr/local/bin`?

Reply:
:eyes: now ...

Reply:
I dislike `/data/jenkins` because I feel that I cannot be sure that I will not be trampled upon regardless of what name I use for directory and script. `/usr/local/bin` looks quite empty and not owned by jenkins. I will move there.

Reply:
Done with commit 1ea634ba15df62cdf187eec83a0a08e792d1cbf6